### DISCUSSION TITLE
Data Panel (單字列表)不同裝置間無法顯示

### DISCUSSION THREAD

**Discussion by ciscteresa** [guopenghui/obsidian-language-learner#161] [Q&A]
請問我的vault是直接存在google雲端硬碟, 當我用另一台電腦/手機開啓雲端硬碟上的vault, data panel無法作用, 是空白的, 不知為何, 請大大可以解惑, 謝謝

**Comment by guopenghui** [comment]
数据保存在本地，可以理解为属于obsidian本身的数据，并不放在vault里面，所以无法随vault一起同步
